worlds/oot_soh/Options.py

```diff
@@ -749,6 +749,13 @@ class SkeletonKey(Toggle):
     display_name = "Skeleton Key"
 
 
+class SlingbowBreakBehives(Toggle):
+    """
+    Both of the Slingshot or Fairy Bow can be used to break beehives if beehives are shuffled.
+    """
+    display_name = "Slingshot/Bow Can Break Beehives"
```

Comment:
```suggestion
    Both the Slingshot and Fairy Bow can break beehives.
    """
    display_name = "Slingshot and Bow Can Break Beehives"
```
Just making it a little more succinct and grammatically correct